tests/integration/replication.tcl

```diff
@@ -1457,3 +1457,52 @@ start_server {tags {"repl external:skip"}} {
         }
     }
 }
+
+foreach disklessload {disabled on-empty-db} {
+    test "Replica should reply LOADING while flushing a large db (disklessload: $disklessload)" {
+        start_server {} {
+            set replica [srv 0 client]
+            start_server {} {
+                set master [srv 0 client]
+                set master_host [srv 0 host]
+                set master_port [srv 0 port]
+
+                $replica config set repl-diskless-load $disklessload
```

Comment:
```suggestion
                $replica config set repl-diskless-load $disklessload
                $replica config set replica-lazy-flush no
```

Reply:
what about adding a similar test for flushing with async.

Reply:
async flush is not blocking, so we won't get -LOADING reply. This tests expects -LOADING reply while doing blocking flush.

Reply:
yes, we can verify it can't be blocked. i'm not sure if it was covered other places.

Reply:
Checking that we do not receive `-LOADING` did not seem reasonable when I wrote the test. 
Instead added a test to check if we increase `lazyfreed_objects`. 

As you mentioned, we don't have a test for `replica-lazy-flush` config. At least, new test will cover it.